examplesTS/00 intro/00 Boilerplate/webpack.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  entry: ['./students.ts'],
+  output: {
+    filename: 'bundle.js',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.ts$/,
+        exclude: /node_modules/,
+        loader: ['babel-loader',
+                 'awesome-typescript-loader',
```

Comment:
You don't need to include both, take a look at:

https://github.com/Lemoncode/webpack-3.x-by-sample/blob/master/02%20Fx/00%20TypeScript/webpack.config.js